rllib/examples/bare_metal_policy_with_custom_view_reqs.py

```diff
@@ -35,6 +36,8 @@ def get_cli_args():
 
     config = {
         "env": "CartPole-v0",
+        # Use GPUs iff `RLLIB_NUM_GPUS` env var set to > 0.
+        "num_gpus": int(os.environ.get("RLLIB_NUM_GPUS", "0")),
```

Comment:
just confirming, is this the only example that is missing num_gpus config?
all the other tests we updated here have this bit already right?
just don't want to waste that gpu if they get scheduled this way.

Reply:
GPU don't really make sense for A3C, as all gradient computations (the most expensive part of the algo) happen on the workers.

Reply:
I don't think we use the `RLLIB_NUM_GPUS` setting a lot. Only in the CI in 1-2 tests (example scripts).